Fix hard coded compiler, flags and tcl lib version for plugins

Ake Sandgren, 20190823
--- plugins/Make-arch.orig	2016-10-21 23:34:39.000000000 +0200
+++ plugins/Make-arch	2019-08-23 10:45:51.403545042 +0200
@@ -337,17 +337,17 @@
 	"ARCH = LINUXAMD64" \
 	"COPTO = -fPIC -m64 -o " \
 	"LOPTO = -fPIC -m64 -lstdc++ -o " \
-	"CC = gcc" \
-	"CXX = g++" \
+	"CC = $(CC)" \
+	"CXX = $(CXX)" \
 	"DEF = -D" \
-	"CCFLAGS = -m64 -O2 -fPIC -Wall" \
-	"CXXFLAGS = -m64 -O2 -fPIC -Wall" \
-	"TCLLDFLAGS = -ltcl8.5 -ldl" \
+	"CCFLAGS = $(CFLAGS)" \
+	"CXXFLAGS = $(CXXFLAGS)" \
+	"TCLLDFLAGS = $(TCLLDFLAGS)" \
 	"NETCDFLDFLAGS = -lnetcdf " \
 	"AR = ar" \
 	"NM = nm -p" \
 	"RANLIB = touch" \
-	"SHLD = gcc -shared"
+	"SHLD = $(CC) -shared"
 
 LINUXCARMA:
 	$(MAKE) dynlibs staticlibs bins \
